Dockerfile

```diff
@@ -17,22 +17,19 @@ WORKDIR /app
 
 RUN apt-get update && apt-get install -y gcc build-essential libpq-dev musl-dev python3-dev
 
-# Set up venv
-RUN python -m venv /opt/venv
-# Make sure we use the virtualenv:
-ENV PATH="/opt/venv/bin:$PATH"
+ENV POETRY_VERSION=1.5.0
```

Comment:
If we use the Make target to install Poetry, we can remove this line. The version could be overriden by mounting a dotenv file. We could keep `POETRY_VERSION` as a build argument but I don't think that's necessary.

Reply:
Yes, I have removed it from here. It is only present in Makefile now.